### DISCUSSION TITLE
Configure line-length for generated code

### DISCUSSION THREAD

**Discussion by jangalinski** [square/kotlinpoet#1969] [General]
I noticed that when I use `listOf(codeblock,codeblock).joinToCode()` to avoid having newlines for each block (I use this by adding members to an annotaion spec), the generated single line is wrapped at a random whitespace.
This still compiles, but looks ugly. Instead of working with the non-breaaking-dot hack for every format/value, can't I just configure kotlin poet that my max line length is set two 200 and avoid wrapping? I mean, I control the generated code, when I want a break, I can configure it.

Am I missing something? Can you help me out?

**Comment by Egorand** [comment]
This has been asked many times in the past: https://github.com/search?q=repo%3Asquare%2Fkotlinpoet+line+length&type=issues.

**Reply by jangalinski** [reply]
Oh, sorry, must have been using wrong search terms. Thanks for pointing out!
